lib/main/pico-sdk/boards/include/boards/0xcb_helios.h

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Do we want to include boards? And all boards supported by SDK?

Reply:
Probably not... I'll remove them - we can always add them back later if needed.